### PR TITLE
:memo:Translating docs to Simplified Chinese

### PR DESCRIPTION
**Pull request by miaowumiaomiaowu**
#1915 
I have translated "collections.md" and "community.md" into Simplified Chinese. 😊

### CODE REVIEW

**Comment by miaowumiaomiaowu** [comment]
Hi @13579606,

I’ve translated some more documentation into Simplified Chinese. Could you please help review it if you have time?
Thank you so much for your help!😄

File: docs/source/cn/guides/collections.md [hunk 1]
```diff
@@ -0,0 +1,226 @@
+<!--⚠️ 请注意，该文件使用Markdown格式，但包含我们的文档构建器的特定语法（类似与MDX),您的Markdown查看器可能无法正确渲染
+-->
+
+# 集合（Collections）
+
+集合（collection）是 Hub 上将一组相关项目（模型、数据集、Spaces、论文）组织在同一页面上的一种方式。利用集合，你可以创建自己的作品集、为特定类别的内容添加书签，或呈现你想要分享的精选条目。要了解更多关于集合的概念和在 Hub 上的呈现方式，请查看这篇 [指南](https://huggingface.co/docs/hub/collections) 
+
+你可以直接在浏览器中管理集合，但本指南将侧重于如何以编程方式进行管理。
+
+## 获取一个集合
```
Comment (13579606) [inline docs/source/cn/guides/collections.md:10]: 获取集合

File: docs/source/cn/guides/collections.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
+<!--⚠️ 请注意，该文件使用Markdown格式，但包含我们的文档构建器的特定语法（类似与MDX),您的Markdown查看器可能无法正确渲染
+-->
+
+# 集合（Collections）
+
+集合（collection）是 Hub 上将一组相关项目（模型、数据集、Spaces、论文）组织在同一页面上的一种方式。利用集合，你可以创建自己的作品集、为特定类别的内容添加书签，或呈现你想要分享的精选条目。要了解更多关于集合的概念和在 Hub 上的呈现方式，请查看这篇 [指南](https://huggingface.co/docs/hub/collections) 
+
+你可以直接在浏览器中管理集合，但本指南将侧重于如何以编程方式进行管理。
```
Comment (13579606) [inline docs/source/cn/guides/collections.md:8]: 你可以直接在浏览器中管理集合，但本指南将重点介绍如何以编程方式进行管理。

File: docs/source/cn/guides/collections.md [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+  item_id='TheBloke/U-Amethyst-20B-AWQ',
+  item_type='model',
+  position=88,
+  note=None
+)
+```
+
+[`get_collection`] 返回的 [`Collection`] 对象包含以下信息:
+- 高级元数据: `slug`, `owner`, `title`, `description`等。
+- 一个 [`CollectionItem`] 对象列表; 每个项目代表一个模型、数据集、Space 或论文。
```
Comment (13579606) [inline docs/source/cn/guides/collections.md:48]: 每个条目代表一个模型、数据集、Space或论文。

File: docs/source/cn/guides/collections.md [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+
+所有集合条目（items）都保证具有：
+- 唯一的 `item_object_id`: 这是集合项在数据库中的唯一 ID
```
Comment (13579606) [inline docs/source/cn/guides/collections.md:51]: 这是集合条目在数据库中的唯一ID

File: docs/source/cn/guides/collections.md [hunk 5]
```diff
@@ -0,0 +1,226 @@
...
+- 一个 `item_id`: Hub 上底层条目的 ID（模型、数据集、Space、论文）；此 ID 不一定是唯一的，仅当 `item_id` 与 `item_type` 成对出现时才唯一
+- 一个 `item_type`: 如model, dataset, Space, paper
```
Comment (13579606) [inline docs/source/cn/guides/collections.md:53]: 一个 `item_type`: 如`model`, `dataset`, `Space`,` paper`

File: docs/source/cn/guides/collections.md [hunk 6]
```diff
@@ -0,0 +1,226 @@
...
+
+`sort` 参数必须是 `"last_modified"`、`"trending"` 或 `"upvotes"` 之一。`item` 参数接受任意特定条目，例如：
+* `"models/teknium/OpenHermes-2.5-Mistral-7B"`
+* `"spaces/julien-c/open-gpt-rhyming-robot"`
+* `"datasets/squad"`
+* `"papers/2311.12983"`
+
+详情请查看 [`list_collections`] 的参考文档。
+
+## 创建一个新集合
```
Comment (13579606) [inline docs/source/cn/guides/collections.md:104]: 创建新集合

File: docs/source/cn/guides/collections.md [hunk 7]
```diff
@@ -0,0 +1,226 @@
...
+"ICCV 2023"
+>>> collection.owner
+"username"
+>>> collection.url
+'https://huggingface.co/collections/owner/iccv-2023-15e23b46cb98efca45'
+```
+
+## 管理集合中的条目
+
+现在我们有了一个 [`Collection`]，接下来要添加条目并进行组织。
```
Comment (13579606) [inline docs/source/cn/guides/collections.md:132]: 接下来要添加条目并进行管理。

File: docs/source/cn/guides/community.md [hunk 8]
```diff
@@ -0,0 +1,141 @@
...
+
+如果是拉取请求（Pull Request），您可以通过 [`DiscussionWithDetails.diff`] 获取原始的 git diff。拉取请求（Pull Request）的所有提交都列在 [`DiscussionWithDetails.events`] 中。
+
+
+## 以编程方式创建和编辑讨论或拉取请求
+
+[`HfApi`] 类还提供了创建和编辑讨论及拉取请求（Pull Request）的方法。
+您需要一个 [访问令牌](https://huggingface.co/docs/hub/security-tokens) 来创建和编辑讨论或拉取请求（Pull Request）。
+
+建议更改仓库最简单的方法是使用 [`create_commit`] API：只需将 `create_pr` 参数设置为 `True` 。此参数也适用于其他封装了 [`create_commit`] 的方法：
```
Comment (13579606) [inline docs/source/cn/guides/community.md:86]: 在 Hub 上对 repo 提出修改建议的最简单方法是使用 [`create_commit`] API：只需将 `create_pr` 参数设置为 `True` 。此参数也适用于其他封装了 [`create_commit`] 的方法：

File: docs/source/cn/guides/community.md [hunk 9]
```diff
@@ -0,0 +1,141 @@
...
+
+>>> create_pull_request(
+...     repo_id="username/repo-name",
+...     title="Hi from the huggingface_hub library!",
+...     token="<insert your access token here>",
+... )
+DiscussionWithDetails(..., is_pull_request=True)
+```
+
+您可以完全使用 [`HfApi`] 类来管理拉取请求和讨论。例如：
```
Comment (13579606) [inline docs/source/cn/guides/community.md:124]: 您可以使用 [`HfApi`] 类来方便地管理拉取请求和讨论。例如：

**Review by 13579606 — COMMENTED** [review]
Hello @miaowumiaomiaowu , it’s my honor to help you with the review. Here are my suggestions to make the sentences more fluent and precise. Feel free to let me know if there’s anything else I can assist with!😁

**Review by miaowumiaomiaowu — COMMENTED** [review]
@13579606 Thank you so much for taking the time to review my PR and for your valuable feedback. I really appreciate your insights and suggestions! I’ll work on addressing the changes as soon as possible.💖

Thanks again for your help!

**Review by 13579606 — APPROVED** [review]
@miaowumiaomiaowu  Thank you for your contribution! These files look so much better now. If there’s anything else I can help with, please don’t hesitate to let me know~

**Review by hanouticelina — APPROVED** [review]
thanks a lot @miaowumiaomiaowu for this contribution and thanks @13579606 for the review 🤗 looks good to me!

**Pull request merged**